<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1576</width>
    <height>956</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MotorScanMX</string>
  </property>
  <property name="windowIcon">
   <iconset resource="scanmx.qrc">
    <normaloff>:/new/prefix1/Graph1-small.png</normaloff>:/new/prefix1/Graph1-small.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>1</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>9</number>
      </property>
      <widget class="QWidget" name="setup" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>X Axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Axis" name="xAxis" native="true"/>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="scan2D">
          <property name="text">
           <string>Y Axis (2-dimensional scan)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Axis" name="yAxis" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="after">
          <property name="toolTip">
           <string>Where the positioners are moved after the scan is complete</string>
          </property>
          <item>
           <property name="text">
            <string>End position</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Start position</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Prior position</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="signalsList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="signalsC">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>254</width>
             <height>755</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QGridLayout" name="signalsL">
               <property name="margin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>1</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_17">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>PV</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Value</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addSignal">
          <property name="toolTip">
           <string>Add signal to be scanned</string>
          </property>
          <property name="text">
           <string>Add signal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="browseSaveDir">
             <property name="toolTip">
              <string>Browse for the directory to save scan data into.</string>
             </property>
             <property name="text">
              <string>to</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="saveDir">
             <property name="toolTip">
              <string>Directory where the scan data are saved.</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="saveName">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Name of the file with the scanned data.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoName">
             <property name="toolTip">
              <string>Automatically construct the filename for the data to be saved.</string>
             </property>
             <property name="text">
              <string>Auto</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="data" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth">
           <number>9</number>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QTableWidget" name="dataTable">
           <property name="columnCount">
            <number>1</number>
           </property>
           <column>
            <property name="text">
             <string>X Axis</string>
            </property>
           </column>
          </widget>
          <widget class="QMdiArea" name="plots">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="background">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>35</red>
              <green>38</green>
              <blue>42</blue>
             </color>
            </brush>
           </property>
           <property name="viewMode">
            <enum>QMdiArea::TabbedView</enum>
           </property>
           <property name="documentMode">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="printResult">
             <property name="toolTip">
              <string>Print the plot.</string>
             </property>
             <property name="text">
              <string>Print</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveResult">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>Save the data to file</string>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qtiResults">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Open with QtiPlot</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%v of %m (%p%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Start / Stop the scan.</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Axis</class>
   <extends>QWidget</extends>
   <header>axis.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="scanmx.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>scan2D</sender>
   <signal>toggled(bool)</signal>
   <receiver>yAxis</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>6</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scan2D</sender>
   <signal>toggled(bool)</signal>
   <receiver>yAxis</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
